Fix timer log output (#54)
nasa/cumulus-message-adapter-js: CHANGELOG.md

and this project adheres to [Semantic
Versioning](http://semver.org/spec/v2.0.0.html).

## [v2.0.4] 2021-12-07

### Fixed

- **CUMULUS-2745**
  - Fixed erroneous log output when running as a lambda incorrectly indicating timeout behaviors were not enabled

>   - Fix streaming interface issue where lambda failure results
>     in an empty line being sent to the CMA resulting in a 'blank'
>     command (')

Reviewer: ```suggestion
  - Fix streaming interface issue where an empty line is being sent to the CMA 
  where it was being interpreted as empty string for the command
```
